src/app/_utils/handleError.ts

```diff
@@ -0,0 +1,36 @@
+import { toast } from "react-toastify";
+import { isClerkAPIResponseError } from "@clerk/nextjs/errors";
+
+const handleClerkError = (error: unknown) => {
```

Comment:
try this 
`const handleClerkError = (error: ClerkAPIError | undefined) => {
  const errorMessage = error?.longMessage ?? "Unknown Clerk error occurred.";
  toast.error(`${errorMessage}`);
};`

`const handleError = (error: unknown) => {
  if (isClerkAPIResponseError(error)) {
    handleClerkError(error.errors[0]);
  } else {
    handleGeneralError(error); // Fallback for non-Clerk errors
  }
};`